roles/consensus/vars/ufw_rules.yml

```diff
@@ -0,0 +1,45 @@
+---
+consensus_ufw_rules:
+  validator:
```

Comment:
please create a similar file for execution_ufw_rules for execution role

Reply:
https://github.com/encapsulate-xyz/berachain-ansible/blob/feature/tasks/setup_environment.yml#L62C1-L70C30

what ports need to be opened for validator and fullnode?

Reply:
for validator

execution_p2p_port, execution_network_discovery_port, usually both are same I think
execution_metrics_port for prometheus

for fullnode

validators ports plus execution_json_rpc_port